app/services/katello/candlepin/consumer.rb

```diff
@@ -26,7 +26,7 @@ class Consumer
 
       attr_accessor :uuid, :owner_label
 
-      def initialize(uuid, owner_label)
+      def initialize(uuid, owner_label = nil)
```

Comment:
1) some of the functions of this 'service' require an owner_label and may fail in unexpected ways if its not passed in.  would it be possible to look it up and pass it in rather than allow a null value?

2) mind writing a test for the action that calls this? :)